docs/signals.md

```diff
@@ -0,0 +1,32 @@
+
+# Signals
+
+Signals are fired before and after the following events which can be found in the file 'constants.py': 
+    - When a version is created the operation sent is 'operation_draft'
+    - When a version is archived the operation sent is 'operation_archive'
+    - When a version is published the operation emitted is 'operation_publish'
+    - When a version is un-published the operation emitted is 'operation_unpublish'
+
+## How To
+
+### Using the version publish and un-publish signal for a CMS Page
+
+The CMS used to provide page publish and unpublish signals which have since been removed. To replicate the behaviour you can listen to changes on the cms model PageContent as shown in the example below.
```

Comment:
I think it would be better to be more specific here - i.e. specify that the CMS provided these signals before version 4.0.